theano/scan_module/scan_checkpoint.py

```diff
@@ -75,7 +75,7 @@ def outer_step(*args):
                                   n_steps=o_n_steps, allow_gc=True)
                                   
     # Keep only the last timestep of every output but keep all the updates
-    return results, updates
+    return results, updates # TODO is it a bug?
```

Comment:
IRL discussion with @carriepl, he confirmed that my interpretation was good.

So remove the TODO and the old code. Update the doc if needed.